apps/devhub/templates/devhub/api/key.html

```diff
@@ -34,27 +35,11 @@
           </ul>
           <p>
           {% trans
-              jwt_url='https://self-issued.info/docs/draft-ietf-oauth-json-web-token.html',
-              node_jwt_url='https://github.com/auth0/node-jsonwebtoken' %}
+              docs_url='http://olympia.readthedocs.org/en/latest/topics/api/auth.html',
+              jwt_url='https://self-issued.info/docs/draft-ietf-oauth-json-web-token.html' %}
             To make API requests, send a <a href="{{ jwt_url }}">JWT (JSON Web Token)</a> as the authorization header.
```

Comment:
I think you're technically supposed to put the acronym in the parentheses like `JSON Web Token (JWT)`.